queries.sql

```diff
@@ -1,23 +0,0 @@
--- Find all animals whose name ends in "mon".
-SELECT * FROM animals WHERE upper(name) LIKE '%MON';
```

Comment:
- [ ] You have created the file with the correct code but removed them and submitted the empty one. Insert these codes again and resubmit the PR. 👍